go.mod

```diff
@@ -7,8 +7,8 @@ require (
 	github.com/google/go-cmp v0.5.5
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
-	github.com/prometheus/common v0.31.0
-	github.com/prometheus/prometheus v2.5.0+incompatible
+	github.com/prometheus/common v0.32.1
+	github.com/prometheus/prometheus v0.0.0-20210315130517-21a282fabecb
```

Comment:
This should be updated in master and then backported to release-0.6.

Reply:
I have opened another pr to solve this issue
https://github.com/kubernetes-sigs/metrics-server/pull/1124